src/js/controller/instream-html5.js

```diff
@@ -145,6 +145,10 @@ define([
                 provider.mute(_model.get('mute'));
 
                 _adModel.on('change:state', changeStateEvent, _this);
+            } else if(provider.state === 'playing') {
```

Comment:
Changing the provider state may cause other issues, although it will fix this particular bug.
Got a second opinion from Donato - When load is called, the state changes to 'loading'. We want to know why the state is playing in this bug, and fix it there.